test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/FileExtensionsAttributeAdapterTest.cs

```diff
@@ -43,33 +77,38 @@ public void AddValidation_WithoutLocalization(string extensions)
             Assert.Collection(
                 context.Attributes,
                 kvp => { Assert.Equal("data-val", kvp.Key); Assert.Equal("true", kvp.Value); },
-                kvp => { Assert.Equal("data-val-fileextensions", kvp.Key); Assert.Equal(expectedMessage, kvp.Value); },
-                kvp => { Assert.Equal("data-val-fileextensions-extensions", kvp.Key); Assert.Equal(extensions.ToLowerInvariant(), kvp.Value); });
+                kvp => { Assert.Equal("data-val-fileextensions", kvp.Key); Assert.Equal(expectedErrorMessage, kvp.Value); },
+                kvp => { Assert.Equal("data-val-fileextensions-extensions", kvp.Key); Assert.Equal(expectedExtensions, kvp.Value); });
         }
 
-        [Fact]
+        [Theory]
+        [InlineData("jpg", "jpg")]
+        [InlineData(".jpg", "jpg")]
+        [InlineData("jpg,png", "jpg,png")]
+        [InlineData("jpg, png", "jpg,png")]
+        [InlineData("JPG, Png", "jpg,png")]
+        [InlineData(".jpg,.png", "jpg,png")]
+        [InlineData(".jpg, .png", "jpg,png")]
```

Comment:
Why is this data set different from the one for `AddValidation_WithoutLocalizationAndCustomFileExtensions(...)`? Suggest adding a `TheoryData<string, string>` property and using it in both tests.